cmd/soroban-rpc/lib/preflight/src/fees.rs

```diff
@@ -132,7 +141,15 @@ fn calculate_host_function_soroban_resources(
         .context("cannot convert storage footprint to ledger footprint")?;
     let read_bytes: u32 = ledger_changes
         .iter()
-        .map(|c| c.encoded_key.len() as u32 + c.old_entry_size_bytes)
+        .map(
+            |c| {
+                let mut size = c.encoded_key.len() as u32 + c.old_entry_size_bytes;
```

Comment:
Ah I missed this - We  no longer need to count the key. @2opremio should I open a follow up PR?

Reply:
Yeah, I was aware. I will fix it later on (it's not critical for the release since it will only add some overhead in the resource estimation)

Reply:
But, if you want to open a PR, by all means! I was assuming you have enough on your plate already.

Reply:
I'll leave it then since it's not critical.